wdn/templates_6.0/examples/card.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dialog</title>
```

Comment:
This should be card as link and not dialog. I would also probably rename the file to `cardAsLink.html`